frontend/packages/dev-console/integration-tests/features/Knative/channel-ODC-3225.feature

```diff
@@ -0,0 +1,148 @@
+    And user clicks on the Create button
+    Then user will see the channel created
+
+
+@regression
+Scenario: Sink event source to Channel
+    Given user is at Developer Perspective
+    When user creates the channel 
+    And user goes to +Add page
+    And user clicks on the Event Source card
+    And user selects Ping Source card
+    And user enters Schedule
+    And user selects the channel from Resource dropdown
+    And user selects the Application
+    And user enters name of the event source
+    And user clicks on the Create button
+    Then user will see that event source is connected to channel
+
+
+@regression
+Scenario: Context Menu for Channel
+    Given user is at Developer Perspective
+    When user creates the channel
+    And user right clicks on the channel
+    Then user will see the Context Menu for channel
+
+
+@regression
+Scenario: Edit Application Groupings action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Application Groupings
+    And user will click on the Application dropdown on the modal
+    And user selects the Application
+    And user clicks on Save button
+    Then user will see the changed Application Groupings of Channel
+
+
+@regression
+Scenario: Edit Application Groupings to unassigned action on Channel
+    Given user is at Developer Perspective
+    And user is having a Channel inside an applicaiton group on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Application Groupings
+    And user will click on the Application dropdown on the modal
+    And user selects the unassigned item
+    And user clicks on Save button
+    Then user will see that Channel is without an application group
+
+
+@regression
+Scenario: Edit Labels action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Labels
+    And user adds the label
+    And user clicks on the Save button on the modal to save labels and close the modal
+    And user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Labels
+    Then user will see the newly added label
+
+
+@regression
+Scenario: Edit Annotations action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Annotations
+    And user adds the annotations
+    And user clicks on the Save button on the modal to save annotation and close the modal
+    And user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Annotations
+    Then user will see the newly added annotation
```

Comment:
Instead on verifying the newly added labels via edit Annotations, we could verfiy the newly added labels/annotations in sidebar details tab. WDYT @makambalaji ?